frontend/app/routes/_public+/apply+/$id+/partner-information.tsx

```diff
@@ -133,14 +138,19 @@ export default function ApplyFlowApplicationInformation() {
           defaultValue={defaultValues.socialInsuranceNumber}
           errorMessage={errorMessages.socialInsuranceNumber}
         />
-        <fieldset>
+        <fieldset id="dateOfBirth">
           <legend className="mb-2 font-semibold">{t('partner-information.dob')}</legend>
           <div className="flex flex-col gap-6 sm:flex-row">
-            <InputSelect id="month" label={t('partner-information.month')} options={monthOptions} name="month" errorMessage={errorMessages.month} />
-            <InputField id="day" label={t('partner-information.day')} name="day" type="number" min={1} max={31} errorMessage={errorMessages.day} />
-            <InputField id="year" label={t('partner-information.year')} name="year" type="number" min={1900} errorMessage={errorMessages.year} />
+            <InputSelect id="month" label={t('partner-information.month')} options={monthOptions} name="month" />
+            <InputField id="day" label={t('partner-information.day')} name="day" type="number" min={1} max={31} />
+            <InputField id="year" label={t('partner-information.year')} name="year" type="number" min={1900} />
           </div>
         </fieldset>
+        {errorMessages.dateOfBirth && (
+          <InputError id="date-of-birth-error" className="mt-2">
```

Comment:
To ensure accessibility, the ID should be designated as "aria-errormessage" for the corresponding inputs, with the inputs themselves having "aria-invalid" set to true.